sql/hive/src/test/scala/org/apache/spark/sql/hive/QueryPartitionSuite.scala

```diff
@@ -68,82 +68,4 @@ class QueryPartitionSuite extends QueryTest with SQLTestUtils with TestHiveSingl
       sql("DROP TABLE IF EXISTS createAndInsertTest")
     }
   }
-
-  test("SPARK-13709: reading partitioned Avro table with nested schema") {
```

Comment:
Moved it to `VersionSuite` to ensure all the Hive metastore versions can correctly infer the schema.